src/Samples/EventBindingsAndBehaviors.Views/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using Elmish.WPF.Samples.EventBindingsAndBehaviors;
+using static Elmish.WPF.Samples.EventBindingsAndBehaviors.Program;
+
+namespace EventBindingsAndBehaviors.Views {
+  public static class Program {
+    [STAThread]
+    public static void Main() =>
+      main(new MainWindow());
+  }
+}
```

Comment:
I think I prefer `using Elmish.WPF.Samples.EventBindingsAndBehaviors;` and `Program.main(...)`. Is that fine with you?

Reply:
The compiler isn't fine with it 😛 

In your code, `Program` resolves to the type on line 6 (i.e. to the `static class` named `Program` in `Program.cs` instead of the "`static class`"/`module` named `Program` in `Program.fs`).

I am have the `using static` line in my application at work, but I also name the two program files `ProgramCSharp.cs` and `ProgramFSharp.fs` to improve clarity.  If we made that change first, then we could write `ProgramFSharp.main(...)` as you suggested.

I am fine with whatever you prefer here.

Reply:
Let's keep the current approach.